aiplatform/api/AIPlatform.Samples/MultimodalMultiImage.cs

```diff
@@ -49,60 +46,30 @@ public class MultimodalMultiImage
         ByteString christRedeemer = await ReadImageFileAsync(
             "https://storage.googleapis.com/cloud-samples-data/vertex-ai/llm/prompts/landmark3.png");
 
-        // Initialize request argument(s)
-        var content = new Content
-        {
-            Role = "USER"
-        };
-        content.Parts.AddRange(new List<Part>()
+        var generateContentRequest = new GenerateContentRequest
         {
-            new()
-            {
-                InlineData = new()
-                {
-                    MimeType = "image/png",
-                    Data = colosseum
-
-                }
-            },
-            new()
-            {
-                Text = "city: Rome, Landmark: the Colosseum"
-            },
-            new()
+            Model = $"projects/{projectId}/locations/{location}/publishers/{publisher}/models/{model}",
+            Contents =
             {
-                InlineData = new()
+                new Content
                 {
-                    MimeType = "image/png",
-                    Data = forbiddenCity
-                }
-            },
-            new()
-            {
-                Text = "city: Beijing, Landmark: Forbidden City"
-            },
-            new()
-            {
-                InlineData = new()
-                {
-                    MimeType = "image/png",
-                    Data = christRedeemer
+                    Role = "USER",
+                    Parts =
+                    {
+                        new Part { InlineData = new() { MimeType = "image/png", Data = colosseum }},
+                        new Part { Text = "city: Rome, Landmark: the Colosseum" },
+                        new Part { InlineData = new() { MimeType = "image/png", Data = forbiddenCity }},
+                        new Part { Text = "city: Beijing, Landmark: Forbidden City"},
+                        new Part { InlineData = new() { MimeType = "image/png", Data = christRedeemer }}
+                    }
                 }
             }
-        });
-
-        var generateContentRequest = new GenerateContentRequest
-        {
-            Model = $"projects/{projectId}/locations/{location}/publishers/{publisher}/models/{model}"
         };
-        generateContentRequest.Contents.Add(content);
 
-        // Make the request, returning a streaming response
```

Comment:
Were the comments here and at line 105 deliberately removed?

Reply:
(Ditto other samples)

Reply:
Yes. I only kept comments in the GeminiQuickstart but the rest wasn't really adding much value.